content/en/profiler/enabling/java.md

```diff
@@ -131,7 +132,22 @@ java \
 {{% /tab %}}
 {{< /tabs >}}
 
-    **Note**: The `-javaagent` argument needs to be before `-jar`, adding it as a JVM option rather than an application argument. For more information, see the [Oracle documentation][6]:
+{{% collapse-content title="Build and run native-image" level="h4" %}}
```

Comment:
Curious what you think, but I think it might flow better if we move the GraalVM collapsed section to around line 158, between the example and step 4? That way the note about `-javagent` more closely relates to step 3.

Also, I think we could change the collapsed section title to include `(Optional)` at the beginning.

Reply:
I was considering this - the downside is that the `-javaagent` note actually applies also to the GrallVM native-image setup. It's just that it isn't explicit.
TBH, I would leave this up to you - if you feel that it flows better, let's do it, I have no strong opinion about that.

Regarding `(Optional)` - sure, that's perfectly fine.

Will you do the adds? You can directly push to this branch ...

Reply:
Added everything (left the section where it was given that the note should apply to both)!